tests/util/mod.rs

```diff
@@ -12,6 +12,8 @@ use log::{error, warn};
 use mio::event::Event;
 use mio::{Events, Interests, Poll, Token};
 
+pub const TIMEOUT: Option<Duration> = Some(Duration::from_millis(500));
```

Comment:
Why do we need such a large timeout? This makes `expect_no_events` take 10 times longer.

I'm also not the biggest fan of one timeout fits all, it's often not the case.

Reply:
Good point. It will be changed back